src/zope/app/form/browser/tests/test_functional_i18n.py

```diff
@@ -62,23 +64,60 @@ class IFieldContent(Interface):
         required=False
         )
 
-
+@implementer(IFieldContent)
 class FieldContent(Persistent):
-    implements(IFieldContent)
+    pass
 
 
 checker = renormalizing.RENormalizing([
-    (re.compile(r"HTTP/1\.1 200 .*"), "HTTP/1.1 200 OK"),
+    (re.compile(r"HTTP/1\.0 200 .*"), "HTTP/1.1 200 OK"),
     ])
 
+class _FakeResponse(FakeResponse):
+    if str is bytes:
+        # py2 has a bug, assuming headers are in unicode already, or
+        # are decodable implicitly to ascii
```

Comment:
I don't have one filed there yet. But I can at least leave a version number for now.